packages/react/src/components/pickers/PeoplePicker/PeoplePickerItems/PeoplePickerItem.styles.ts

```diff
@@ -127,8 +140,16 @@ export function getStyles(props: IPeoplePickerItemSelectedStyleProps): IPeoplePi
       invalid &&
         selected &&
         !disabled && {
-          background: palette.redDark,
+          selectors: {
+            ':focus-within': {
+              background: palette.redDark,
+              color: palette.white,
+            },
+          },
```

Comment:
_nit:_ I think you can just use the selectors directly without having to specify `selectors` anymore, in which case removing it would be a small bundle size saving

```suggestion
          ':focus-within': {
            background: palette.redDark,
            color: palette.white,
          },
```

Reply:
done!